src/storage/src/monitor/local_metrics.rs

```diff
@@ -85,8 +90,10 @@ impl StoreLocalStatistic {
                 .inc_by(self.bloom_filter_might_positive_count);
         }
 
-        if self.remote_io_time > 0.0 {
-            metrics.remote_read_time.observe(self.remote_io_time);
+        if self.remote_io_time.load(Ordering::SeqCst) as f64 > 0.0 {
```

Comment:
we only need load once because the operation `report` will always be called after all IO operation end.

```rust
let t = self.remote_io_time.load(Ordering::Relaxed);
if t > 0 {
  metrics.remote_read_time.observe(t as f64);
}
```